docs/_pages/releases.md

```diff
@@ -15,6 +15,7 @@ sidebar:
 * Added `collectionOfStrings.Should().NotContainMatch()` to assert that the collection does not contain a string that matches a wildcard pattern 
 * The `Using`/`When` option on `BeEquivalentTo` will now use the conversion rules when trying to match the predicate.
 * Added `NotBeWritable` to `PropertyInfoSelectorAssertions` to be able to assert that properties are not writable.
+* Added extension to assert `TaskCompletionSource<T>`
```

Comment:
Add link to this PR.
I have a PR coming up to fix it for the other bullet points.